src/GitVersion.Core/Git/ICommit.cs

```diff
@@ -2,6 +2,8 @@ namespace GitVersion;
 
 public interface ICommit : IEquatable<ICommit?>, IComparable<ICommit>, IGitObject
 {
+    public bool IsMergeCommit => Parents.Count() >= 2;
+
```

Comment:
I think public is redundant here. Please also remove the empty line to be consistent with other interfaces in the Git Folder

Reply:
Yes you are right. Unfortunately the design in .net is not consistent compared to a definition in classes because there it defaults to internal.